src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/queue/DataToSend.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound.queue;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hl7.fhir.dstu3.model.Parameters;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.TransactionType;
+
+public class DataToSend {
+
+    @Getter
+    private Object content;
+    @Getter
+    @Setter
+    private String operationId;
+    @Getter
+    @Setter
+    private TransactionType transactionType;
+
+    public DataToSend setContent(final Parameters parameters) {
+        this.content = parameters;
+        return this;
+    }
+}
```

Comment:
The annotations can be applied to the class here. Lombok will see that there's already a method called `setContent` and won't try to replace it.

```suggestion
@Getter
@Setter
public class DataToSend {

    private Object content;

    private String operationId;

    private TransactionType transactionType;

    public DataToSend setContent(final Parameters parameters) {
        this.content = parameters;
        return this;
    }
}
```

Reply:
Good spot!